tests/test_configuration.py

```diff
@@ -799,6 +824,24 @@ def setUpClass(cls):
 	def tearDownClass(cls):
 		sys.path.pop(0)
 
```

Comment:
Not enough blank lines. I really suggest using syntastic or the like if you are not using Vim.